### PR TITLE
Volunteer form vue version with APIs

### PR DESCRIPTION
**Pull request by arpavlic03**
This is the volunteer form that will be used instead of the Airtable form. Mark designed the Amazon APIs for uploading photos into a S3 bucket and for creating a volunteer into airtable with the url supplied by the Amazon S3 bucket. UI and logic is programmed in Vuejs

### CODE REVIEW

File: _includes/volunteer/main.html [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+            <option value="South Dakota">South Dakota</option>
+            <option value="Tennessee">Tennessee</option>
+            <option value="Texas">Texas</option>
+            <option value="Utah">Utah</option>
+            <option value="Virginia">Virginia</option>
+            <option value="Vermont">Vermont</option>
+            <option value="Washington">Washington</option>
+            <option value="Wisconsin">Wisconsin</option>
+            <option value="West Virginia">West Virginia</option>
+            <option value="Wyoming">Wyoming</option>
```
Comment (seanlawrenz) [inline _includes/volunteer/main.html:129]: Are the accepting values the state's full name? I hope you copied this from somewhere. That's a lot of typing.

File: _includes/volunteer/main.html [hunk 2]
```diff
@@ -0,0 +1,297 @@
+<div class="jumbotron background-white">
+<form id="volunteer-form" name="volunteer" enctype="multipart/form-data" v-on:submit.prevent="addVolunteer"> 
+  <div class = "row">
+    <div class="col-md-7 col-lg-5">
+      <div class="form-group">
+        <input type="text" id="firstName"  v-model="newVolunteer.firstName" required class="form-control volunteerField">
+        <label for="firstName" :class="{'stateField': newVolunteer.firstName}" class="form-control-placeholder">First Name</label>
```
Comment (seanlawrenz) [inline _includes/volunteer/main.html:7]: The `for` attribute on the `label` works with the `name` attribute on `input`. We'll have to add the `name` attributes. :(

File: assets/js/volunteer.js [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+      }
+
+      axios
+        .post(
+          `https://wduc7ys73l.execute-api.us-east-1.amazonaws.com/dev/volunteers`,
+          vol,
+          config
+        )
+        .then((response) => {
+          if (response.status == "200") {
```
Comment (seanlawrenz) [inline assets/js/volunteer.js:153]: Is the response status a string?

File: _includes/volunteer/main.html (hunk 1 above)
Comment (cah-markharris) [reply _includes/volunteer/main.html:129]: yeah. the accepting values are the state's full name. It's what's in Airtable. And yeah, it's a lot of typing. Took about 5-10 min.

File: assets/js/volunteer.js (hunk 3 above)
Comment (cah-markharris) [reply assets/js/volunteer.js:153]: good question. actually this is probably not needed. if there's any error, the 'catch' executes. response.status is expected to be 200 if we make it into the "then".

**Pull request merged**